src/main/java/net/dancier/domain/Recommendation.java

```diff
@@ -0,0 +1,32 @@
+package net.dancier.domain;
+
+import net.dancier.domain.dance.Dancer;
+import net.dancier.domain.dance.School;
+
+public class Recommendation<T extends Recommendable> {
+
+    public enum Type {
+        DANCER,
+        SCHOOL,
+        WORKSHOP,
+        EVENING,
+        ETC
```

Comment:
What is this? Should be documented. Is this just "everything that is not defined yet". Should this be handled at all? (Personally I would throw an exception in such a case I think)

Reply:
I decided to follow your personal preference by throwing an exeption.